### PR TITLE
Allow multiple iterations for load and session creation and initial bind and evaluate in winmlrunner static library

### PR DESCRIPTION
**Pull request by ryanlai2**
This change allows multiple iterations for loading a model and also multiple iterations for session creation  for WinMLRunner static library.

The purpose of this change is for performance measuring, it is better to run load / session creation / first bind / first eval multiple times to get stability.

**NOTE:** This PR changes the schema of the performance file.

### CODE REVIEW

File: Tools/WinMLRunner/src/Run.cpp [hunk 1]
```diff
@@ -799,39 +831,48 @@ int run(CommandLineArgs& args, Profiler<WINML_MODEL_TEST_PERF>& profiler) try
...
+                                {
+                                    const std::wstring& inputImagePath = L"";
                                     RunConfiguration(args, output, session, lastHr, model, deviceType, inputBindingType,
                                                      inputDataType, winrtDevice, deviceCreationLocation, profiler, path,
-                                                     inputImagePath);
+                                                     L"", sessionCreationIteration);
                                 }
+                                // Close and destroy session
+                                session.Close();
+                                session.~LearningModelSession();
```
Comment (youngkim93) [inline Tools/WinMLRunner/src/Run.cpp]: This is probably not necessary

File: Tools/WinMLRunner/src/Run.cpp [hunk 2]
```diff
@@ -799,39 +831,48 @@ int run(CommandLineArgs& args, Profiler<WINML_MODEL_TEST_PERF>& profiler) try
...
                                 // Resets all values from profiler for bind and evaluate.
                                 profiler.Reset(WINML_MODEL_TEST_PERF::BIND_VALUE, WINML_MODEL_TEST_PERF::COUNT);
                             }
-                            if (args.IsImageInput())
+                            for (uint32_t sessionCreationIteration = 0;
+                                sessionCreationIteration < args.NumSessionCreationIterations();
+                                sessionCreationIteration++)
                             {
-                                for (const std::wstring& inputImagePath : args.ImagePaths())
+                                lastHr = CreateSession(session, winrtDevice, model, args, output, deviceType,
```
Comment (youngkim93) [inline Tools/WinMLRunner/src/Run.cpp]: As we have discussed, it's probably better to keep CreateSession outside of this nested for loops, and in the function CreateSession, you can create a session, load and eval for each input types

File: Tools/WinMLRunner/src/OutputHelper.h [hunk 3]
```diff
@@ -767,72 +767,158 @@ class OutputHelper
     void WritePerformanceDataToCSV(const Profiler<WINML_MODEL_TEST_PERF>& profiler, int numIterations,
                                    std::wstring model, const std::string& deviceType, const std::string& inputBinding,
                                    const std::string& inputType, const std::string& deviceCreationLocation,
-                                   const std::vector<std::pair<std::string,std::string>>& perfFileMetadata) const
+                                   const std::vector<std::pair<std::string, std::string>>& perfFileMetadata) const
     {
-        double loadTime = profiler[LOAD_MODEL].GetAverage(CounterType::TIMER);
-        double createSessionTime = profiler[CREATE_SESSION].GetAverage(CounterType::TIMER);
+        double averageLoadTime = profiler[LOAD_MODEL].GetAverage(CounterType::TIMER);
```
[the remark this answers is not in the document]
Comment (ryanlai2) [reply Tools/WinMLRunner/src/OutputHelper.h]: Yes that would make it more clean. I will do this in a future commit

File: Tools/WinMLRunner/src/OutputHelper.h [hunk 4]
```diff
@@ -868,11 +954,33 @@ class OutputHelper
                      << ","
                      << "iterations"
                      << ","
-                     << "load (ms)"
+                     << "load iterations"
```
[the remark this answers is not in the document]
Comment (ryanlai2) [reply Tools/WinMLRunner/src/OutputHelper.h]: Yes that would make it more clean. I will do this in a future commit

File: Tools/WinMLRunner/src/Run.cpp (hunk 1 above)
Comment (ryanlai2) [reply Tools/WinMLRunner/src/Run.cpp]: Will remove

**Pull request merged**